target/linux/ath79/image/generic.mk

```diff
@@ -3307,6 +3307,19 @@ define Device/yuncore_xd3200
 endef
 TARGET_DEVICES += yuncore_xd3200
 
+define Device/yuncore_cpe830
+  SOC := qca9533
+  DEVICE_VENDOR := YunCore
+  DEVICE_MODEL := CPE830(D)
+  DEVICE_ALT0_VENDOR = KuWfi
+  DEVICE_ALT0_MODEL = CPE830(D)
+  IMAGE_SIZE := 16000k
+  IMAGES += tftp.bin
+  IMAGE/tftp.bin := $$(IMAGE/sysupgrade.bin) | yuncore-tftp-header-16m
+  DEVICE_PACKAGES := rssileds kmod-leds-gpio kmod-ledtrig-gpio kmod-ledtrig-netdev kmod-ath9k -swconfig
```

Comment:
`kmod-ath9k` is already included. `kmod-ledtrig-gpio` seems not necessary. And I can't find package `kmod-ledtrig-netdev`?
https://git.openwrt.org/?p=openwrt%2Fopenwrt.git&a=search&h=f71ce1c34e43a1722eeadefa89324b23d0463022&st=grep&s=ledtrig-netdev

Reply:
@DragonBluep Not sure where ath9k is already included ?
Others are needed for the led 
kmod-ledtrig-netdev is in 24.10

Reply:
`kmod-leds-gpio` will provide a trigger event from GPIO. we don't need it here.

Reply:
Done